draft-tus-resumable-uploads-protocol.md

```diff
@@ -257,7 +261,7 @@ The request MUST use the `DELETE` method and include the `Upload-Token` header.
 
 If the server has successfully released the resources allocated for this token, it MUST send back a `204 (No Content)` response.
 
-The server MUST terminate any ongoing Upload Transfer Procedure for the same token before sending the response.
+The server MAY terminate any ongoing Upload Transfer Procedure for the same token before sending the response by abruptly terminating the HTTP connection or stream.
```

Comment:
Probably a copy/paste error. Shouldn't this be that the server MUST terminate any ongoing Upload Transfer Procedure before responding but MAY do so by abruptly terminating?

Reply:
Since this entire procedure is optional, I didn't think it's necessary to require the server to do this. It doesn't hurt that much if the server doesn't terminate ongoing transfers here.( Unlike the other places where allowing parallel transfers to continue may corrupt the upload.)

Reply:
That makes sense. Thanks for clarifying.